modules/vstudio/vs200x_vcproj.lua

```diff
@@ -1264,7 +1264,7 @@
 
 
 	function m.omitDefaultLib(cfg)
-		if cfg.flags.OmitDefaultLibrary then
+		if cfg.flags.OmitDefaultLibrary or cfg.nodefaultlib == "On" then
```

Comment:
Does the `cfg.flags.OmitDefaultLibrary` check need to remain?

Reply:
Looks like I missed this one. Removing.